CharlieBackend.Panel/Views/Calendar/Index.cshtml

```diff
@@ -70,6 +89,21 @@
             $("#deleteEventOccurrence").attr('href', "/EventOccurrence/DeleteEventOccurrence/" + seEventOccurrenceId);
         });
     </script>
+
+    <script type="text/javascript">
+        function ChangeWeekFormat() {
+            var x = document.getElementById("weekFormat").value;
+
+            if (x == 1) {
+                $(".weekend").removeClass("col-calendar-full-week").addClass("d-none");
+                $(".col-calendar-full-week").removeClass("col-calendar-full-week").addClass("col-calendar-working-week");
+            }
+            else if (x == 2) {
+                $(".col-calendar-working-week").removeClass("col-calendar-working-week").addClass("col-calendar-full-week");
+                $(".weekend").removeClass("d-none").addClass("col-calendar-full-week");
+            }
+        }
+    </script>
```

Comment:
same as css, I don't think that we should have js code there, we have this file for example to store js script and then just include it from ~\WhatBackend\CharlieBackend.Panel\wwwroot\js\sb-admin-2.js